config/OPM-Reconfig.cfg

```diff
@@ -0,0 +1,499 @@
+// Original draft 2019/02/20 by Eridan/HSJasperism
+
+// Removes the Asteroid configs for Kopernicus
+// Kopernicus will still load its ScenarioModule named DiscoverableObjects after killing stock ScenarioDiscoverableObjects
+// but void Start()'s foreach loop will not go though anything because no Asteroid configs will exist.
+
+@Kopernicus:AFTER[OPM]:NEEDS[CustomAsteroids]
+{
+	!Asteroid[Stock] {}
+	!Asteroid[OPMSarnus] {}
+	!Asteroid[OPMUrlum] {}
+	!Asteroid[OPMNeidon] {}
+}
+
+// Custom Asteroid Configs
+// Jool, Sarnus, Urlum, & Neidon configs based on Artifexian's tutorials on Gas Giant systems
+// Sarnus, Urlum, & Neidon attempt to mimic JadOfMaar's configs
+
+AsteroidSets:NEEDS[CustomAsteroids]
```

Comment:
The `:NEEDS` here is a bit redundant. Did you mean `:NEEDS[OPM]`?

Reply:
Agreed